sykepenger-model/src/main/kotlin/no/nav/helse/person/Arbeidsgiver.kt

```diff
@@ -321,16 +321,22 @@ internal class Arbeidsgiver private constructor(
     internal fun håndter(søknad: Søknad) {
         korrigerFerieIForkant(søknad)
         if (Toggles.OppretteVedtaksperioderVedSøknad.enabled) return håndterEllerOpprettVedtaksperiode(søknad, Vedtaksperiode::håndter)
-        søknad.kontekst(this)
-        noenHarHåndtert(søknad, Vedtaksperiode::håndter, "Forventet ikke søknad. Har nok ikke mottatt sykmelding")
-        finalize(søknad)
+        håndterSøknad(søknad, Vedtaksperiode::håndter)
     }
 
     internal fun håndter(søknad: SøknadArbeidsgiver) {
         if (Toggles.OppretteVedtaksperioderVedSøknad.enabled) return håndterEllerOpprettVedtaksperiode(søknad, Vedtaksperiode::håndter)
-        søknad.kontekst(this)
-        noenHarHåndtert(søknad, Vedtaksperiode::håndter, "Forventet ikke søknad til arbeidsgiver. Har nok ikke mottatt sykmelding")
-        finalize(søknad)
+        håndterSøknad(søknad, Vedtaksperiode::håndter)
+    }
+
+    private fun <Hendelse : SykdomstidslinjeHendelse> håndterSøknad(hendelse: Hendelse, håndterer: Vedtaksperiode.(Hendelse) -> Boolean) {
+        hendelse.kontekst(this)
+        if (!noenHarHåndtert(hendelse, håndterer,"Forventet ikke ${hendelse.kilde}. Har nok ikke mottatt sykmelding")) {
```

Comment:
Jeg tror ikke det er vits å sjekke for disse tingene når man håndterer søknad.
Tror disse ble lagt inn dersom vi skulle opprette vedtaksperiode

Reply:
Åja. Skjønner hva du mener. Men hva er grunnen til det egentlig? 
Altså, hvorfor ønsker vi ikke å sjekke `hendelse.forGammel()` og `Utbetaling.harBetalt(utbetalinger, hendelse.periode()))` når vi mottar søknad?

Merk at vi sjekker om hendelsen har errors under på linje 338 for å sende ut et `hendelseIkkeHåndtert`-event. 
Det er noe som har kommet **etter** innføringen av opprettelse av vedtaksperiode fra søknad. 
(Det brukes til datainnsikt-formål og for å kommunisere til spre-oppgaver appen at det skal opprettes gosys-oppgave på søknaden.)

Reply:
`forGammel()` trengs bare sjekke der vi _oppretter_ vedtaksperioder (feks ved sykmelding). Om søknaden er for gammel, så var også sykmeldingen det da vi mottok den. Sjekken blir i så fall overflødig